external-crates/move/crates/move-compiler/src/linters/mod.rs

```diff
@@ -175,6 +176,12 @@ lints!(
         LinterDiagnosticCategory::Complexity,
         "combinable_comparisons",
         "comparison operations condition can be simplified"
+    ),
+    (
+        UnnecessaryPublicEntry,
+        LinterDiagnosticCategory::Style,
+        "public_entry",
+        "functions should either be `public` or `entry`"
```

Comment:
This should be `sui` only

Reply:
Thanks! I explicitly asked it in the PR description, can move it to Sui.